target/linux/ath79/dts/ar7242_engenius_eap350-v1.dts

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7242.dtsi"
```

Comment:
This include needs to be moved _before_ the others (changed in master recently).